### PR TITLE
Add StakePool Operator Scripts builder tool

### PR DESCRIPTION
**Pull request by rphair**
Implementing https://github.com/cardano-foundation/developer-portal/issues/778.

> readme: https://github.com/gitmachtl/scripts/blob/master/cardano/mainnet/README.md

I would like to link to this directly but the `operatortool` keyword won't allow linking anywhere outside the `operate-a-stake-pool` section.

Other keywords considered & not used (let me know if you feel differently):

- `wallet` - it prepares Tx like a wallet but doesn't manage funds.
- `nft` - could use mint/burn scripts for that, but not specifically NFT's.

Note I'm adding in the `cli` keyword after I write this, as well as for `cardano-signer` where we missed it in https://github.com/cardano-foundation/developer-portal/pull/774.

**p.s., edit:** this was added near the beginning because this tool was likewise there at the beginning.

### CODE REVIEW

**Review by katomm — APPROVED** [review]
Oh that was a fast one. Thank you! 

#778

**Comment by rphair** [comment]
quickdraw-default

**Review by rdlrt — APPROVED** [review]

**Comment by gitmachtl** [comment]
@rphair thx! i think 'Serialization' can be added as a keyword. sadly the is no 'HardwareWallet' keyword available and 'Voting'. its a bit hard to categories the SPO scripts, because can be used for many different things.

File: src/data/builder-tools.js [hunk 1]
```diff
@@ -339,6 +339,14 @@ export const Showcases = [
     getstarted: "/docs/get-started/blockfrost",
     tags: ["favorite", "http", "json", "hosted"],
   },
+  {
+    title: "StakePool Operator Scripts",
+    description: "Scripts to help manage your stake pool, send transactions, mint/burn tokens, etc. via the CLI.",
```
Comment (gitmachtl) [inline src/data/builder-tools.js:344]: "Hardware-Ledger/Trezor Support" would be a nice addition in the description.

Comment (rphair) [reply src/data/builder-tools.js:344]: I can see from the preview that there's a lot more space to add the HW Wallet component to the description (I was judging by the typical line length in the JS file, which was misleading about the amount of space available).  I'll submit a revision that that element in it.

**Review by gitmachtl — APPROVED** [review]

**Pull request merged**